README.md

```diff
@@ -22,28 +22,31 @@ Open the repository directory `ACMAS-Frontend` and navigate to the `ACMAS` direc
  directory in the command-line or terminal.
 
 ---
-#### Turn on the Docker containers when you want to start working
+#### __Turn on the Docker containers when you want to start working__
 `docker-compose -f docker-compose.yml up -d --build`
-#### Turn off the Docker containers when you are done working
-`docker-compose -f docker-compose.yml down -v`
+#### __Turn off the Docker containers when you are done working__
+If you want to shut down normally: `docker-compose -f docker-compose.yml down`
+
+If you want to flush your database and project files: `docker-compose -f docker-compose.yml down -v`
 
 ---
 ### Using ACMAS with Docker
 ___After turning on the Docker container follow these steps:___
 #### Ensure that the Docker application was successfully built and is running
 - Successful builds will return no errors (red)
 - One of the following messages will appear in your command prompt
+- __Note:__ If you do not flush, only the affected containers will appear
 
   ```shell
   Running 3/3
   - Container acmas-db-1      Started
   - Container acmas-web-1     Started
   - Container acmas-nginx-1   Started
   ```
-  
+
 - Ensure that the application's containers in Docker Desktop all appear as __green__
-- Navigate to `localhost:1337` in any web browser
-  - If this does not work, default to using `127.0.0.1:1337`
+- Navigate to `localhost` in any web browser
+  - If this does not work, default to using `127.0.0.1`

```

Comment:
I like having the port number at the end of the `localhost` since the user might be running multiple applications. Also, elastic search will be on localhost, with a different port, so being specific here would help.